components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtDBQueries.java

```diff
@@ -147,7 +147,8 @@
     public static final String LOAD_CLAIM_MAPPING_BY_APP_ID = "SELECT IDP_CLAIM, SP_CLAIM, IS_REQUESTED, IS_MANDATORY, DEFAULT_VALUE " +
                                                               "FROM SP_CLAIM_MAPPING WHERE APP_ID = ? AND TENANT_ID = ?";
     public static final String LOAD_CLAIM_MAPPING_BY_APP_NAME = "SELECT IDP_CLAIM, SP_CLAIM, IS_REQUESTED, IS_MANDATORY, DEFAULT_VALUE "
-                                                                + "FROM SP_CLAIM_MAPPING WHERE APP_ID = (SELECT ID FROM SP_APP WHERE APP_NAME = ?) AND TENANT_ID = ?";
+                                                                + "FROM SP_CLAIM_MAPPING WHERE APP_ID = (SELECT ID "
```

Comment:
Better leave this line as it was, as we are not doing any change related to connection here.

Reply:
removed from here and create a new PR for this issue in future